canal/rows.go

```diff
@@ -57,7 +58,14 @@ func (r *RowsEvent) handleUnsigned() {
 			case int16:
 				r.Rows[i][index] = uint16(t)
 			case int32:
-				r.Rows[i][index] = uint32(t)
+				if strings.Contains(strings.ToLower(r.Table.Columns[i].RawType), "mediumint") {
```

Comment:
interesting, can you add some comments here? 

maybe it is better to add a test.

Reply:
Maybe the implementation from [gh-ost](https://github.com/github/gh-ost/blob/d2726c77f86cb65e25bce5c0ac5fe3fa0c997488/go/sql/types.go#L57-L67) is better and more clear?

Reply:
@siddontang added comments and test. 
@NearTan I think so, thanks.

Reply:
Maybe we can do more, to add a similar MediumIntColumnType.
I worry that comparing string every time can reduce the performance.

Reply:
good idea